diff --git a/CMakeLists.txt b/CMakeLists.txt
index 64aa3b8..a2b4b45 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -103,13 +103,6 @@ install(DIRECTORY include/
 # generate a CMake configuration file for consumption by CMake's `find_package`
 # intrinsic
 #
-write_basic_package_version_file("frozenConfigVersion.cmake"
-  VERSION ${frozen_VERSION}
-  COMPATIBILITY SameMajorVersion)
-
-install(FILES "${CMAKE_CURRENT_BINARY_DIR}/frozenConfigVersion.cmake"
-  DESTINATION share/cmake/frozen
-  PERMISSIONS OWNER_READ GROUP_READ)
 
 install(EXPORT frozenConfig
   FILE frozenConfig.cmake
